ClipboardAPI/clipboard-change-event-explainer.md

```diff
@@ -0,0 +1,221 @@
+|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Windows       | We can use the [AddClipboardFormatListener](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-addclipboardformatlistener) function (winuser.h) which posts a [WM_CLIPBOARDUPDATE](https://learn.microsoft.com/en-us/windows/win32/dataxchg/wm-clipboardupdate) message whenever the clipboard changes. |
+| MacOS         | No API provided, need to poll OS clipboard for changes                                                                                                                                                                                                                                                                      |
+| Linux         | TBD                                                                                                                                                                                                                                                                                                                         |
+| ChromeOS      | TBD                                                                                                                                                                                                                                                                                                                         |
+| Android / iOS | TBD                                                                                                                                                                                                                                                                                                                         |
+
+### 6.2 Considered alternative: Given the page focus restrictions, check clipboard hash change when page regains focus
+On page focus event, we can check the current hash of the clipboard (using a [SequenceNumber API](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getclipboardsequencenumber)) and compare it with the previously stored hash to infer if the clipboard changed. This is an alternative to directly using OS provided APIs for monitoring clipboard. For clipboard changes occurring within the browser, we can easily obtain the clipboard change event from within the browser.
+
+#### 6.2.1 Platform support for getting clipboard SequenceNumber/Version number
```

Comment:
Is this also supposed to be a "considered alternative"? If so, can it also be labeled as such?

Reply:
6.2 is already labelled as "considered alternative".  6.2.1 is providing more information for 6.2 hence didn't add "considered alternative" here. Should I still add the label here ?